src/Pages/Login/Login.js

```diff
@@ -1,6 +1,205 @@
-import React from 'react';
+import React, { useState } from 'react';
+import { useHistory, Link } from 'react-router-dom';
+// import axios from 'axios';
 import styled from 'styled-components';
-
+import { useForm } from 'react-hook-form';
+import LoginFooter from './LoginFooter';
 export default function Login() {
-  return <div></div>;
+  const [inputValue, setInputValue] = useState({
+    idValue: '',
+    pwValue: '',
+  });
+
+  const { register, handleSubmit, errors } = useForm();
+
+  const history = useHistory();
+
+  const onSubmit = (data) => {
+    console.log(data);
+  };
+
+  // 구조화
+  const { idValue, pwValue } = inputValue;
+
+  //id, password 입력 값 저장
+  const handleInput = (e) => {
+    const nextInputValue = {
+      ...inputValue,
+      [e.target.name]: e.target.value,
+    };
+    setInputValue(nextInputValue);
+    console.log('inputValue: ', inputValue);
+    console.log('idValue: ', idValue);
+  };
+
+  //master
+  //id:soojsooj
+  //pw:PW1!soojsooj
+
+  //seller
+  // id: seller1
+  //pw: PW1!seller1
+  const goToHome = (e) => {
+    e.preventDefault();
+    fetch('http://10.251.1.180:5000/account/signin', {
+      method: 'POST',
```

Comment:
```suggestion
    e.preventDefault();
    fetch('http://10.251.1.180:5000/account/signin', {
      method: 'POST',
```
지은님 URL은 config.js 만든 파일에서 관리하면 좋을것같아요~!

Reply:
수정했습니다~